### DISCUSSION TITLE
Upgraded from 0.70.0 to 0.70.6, not allowed by rnx-align-deps?

### DISCUSSION THREAD

**Discussion by benomatis** [microsoft/rnx-kit#2086] [Q&A] [answered]
I had `"react-native": "^0.70.0"` that I changed to `"react-native": "^0.70.6"` and ran `yarn install`, but `rnx-align-deps` expects `0.70.0` only, why?

**Comment by tido64** [comment, marked as answer]
This is to ensure that your package manager resolves the requested version, `^0.70.0`, to the same version for your whole repository. If you had, for example, both `^0.70.0` and `^0.70.6`, you _can_ end up with two different copies of react-native. You can find more information about the reasons in this document: https://microsoft.github.io/rnx-kit/docs/architecture/dependency-management

That said, the caret (^) allows you to bump patch version. `^0.70.0` is the same as `0.70.x`. If you want to upgrade to 0.70.6, you can use the upgrade command of your package manager (instructions for [Yarn](https://classic.yarnpkg.com/en/docs/cli/upgrade)). If you prefer to do so manually, you can also try removing the `react-native` entry in `yarn.lock` and re-run install.

**Reply by benomatis** [reply]
That makes perfect sense, thank you. I used `yarn upgrade react-native` and I saw it upgraded to 0.70.6 without the actual full version name showing on `package.json`, there it remained `^0.70.0`, but `yarn.lock` properly shows the version 0.70.6. Thank you.
